cmd/pctl/install.go

```diff
@@ -47,27 +49,44 @@ func installCmd() *cli.Command {
 				Usage: "If given, install will create a PR for the modifications it outputs.",
 			},
 			&cli.StringFlag{
-				Name:        "remote",
+				Name:        "pr-remote",
 				Value:       "origin",
 				DefaultText: "origin",
 				Usage:       "The remote to push the branch to.",
 			},
 			&cli.StringFlag{
-				Name:        "base",
+				Name:        "pr-base",
 				Value:       "main",
 				DefaultText: "main",
 				Usage:       "The base branch to open a PR against.",
 			},
+			&cli.StringFlag{
+				Name:        "pr-branch",
+				Value:       "main",
+				DefaultText: "main",
+				Usage:       "The branch to create the PR from.",
+			},
+
```

Comment:
is it possible to "group" flags with this CLI tool? Would be cool and help readability

Reply:
will check

Reply:
Nope, nothing that would be convenient and grouping. :/